### ISSUE TITLE
Question about the baseline structure compared to iCAN

### ISSUE THREAD

**Issue by Gaoyiminggithub** [DirtyHarryLYL/Transferable-Interactiveness-Network#18]
Hi, thanks for the great work!
I have read your code. it seems your baseline structure of $RC_D$ is the same as iCAN (please correct me if I'm wrong). 
I wonder why the results of $RC_D$ seems lower than the iCAN,  if you could address it, that would be great. 
Thanks very much!

**Comment by DirtyHarryLYL** [comment]
Yeah, we have refined our model and use the 3 streams from iCAN which has useful global attention module, with this modification the C without D becomes better performance than the naive conv+fc structure. Our P is designed as a flexible module which can be cooperated with other HOI models. 
We also tried to use the attention module in our P, but we haven't obtained improvement yet. Maybe a deeper study on the attention structure design will bring a larger improvement.

**Comment by Gaoyiminggithub** [comment]
Thank you for your reply.
I just wonder why on the HICO-DET(or VCOCO), your baseline method $RC_D$ achieve 13.75 mAP (43.2 mAP on VCOCO) while iCAN can achieve 14.84 mAP (44.7 mAP on VCOCO), because it seems your baseline structure is the same as iCAN. (why using the same structure but it achieves a lower performance)
Could you tell me the performance of the baseline $RC_D$  in the reimplementation version?
Thanks very much!

**Comment by DirtyHarryLYL** [comment]
In the paper version, we used a simple structure as seen in the paper figure, roi pooling feature--res--pooling--fcs--cls, this vanilla implementation achieves 13.75 mAP on HICO-DET.
We believe that if we use more powerful HOI classifier C to cooperate with P, we may obtain more performance improvement. Thus we tried to adopt the HOI classifier from iCAN in the reimplementation version,  and we use cosine learning rate scheme instead of normal decay. C without D ($RC_D$) can achieve 15.01 11.16 16.21 16.60 11.72 17.88 mAP on sets of HICO-DET, and 45.61 mAP on V-COCO, which are slightly higher than the scores of iCAN model.

BTW, we also tried some other tricks like some new loss function for long-tail distribution, and refine our pose results, but it seems that the improvements of different modules and tricks may have overlap. The total improvement of the C+D model is small.
